spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/integration/outbound/PubSubMessageHandler.java

```diff
@@ -46,7 +47,7 @@
  *
  * @author João André Martins
  */
-public class PubSubMessageHandler extends AbstractMessageHandler {
+public class PubSubMessageHandler extends AbstractMessageHandler implements Lifecycle {
```

Comment:
What is this new functionality here about?

I don't see any logic in the `start()`, neither `stop()`.

So, what is the point of void implementation?

Reply:
[This test](https://github.com/spring-cloud/spring-cloud-stream/blob/master/spring-cloud-stream-binder-test/src/main/java/org/springframework/cloud/stream/binder/AbstractBinderTests.java#L129) seems to require this change.

Reply:
Then that test is wrong. That is not our code responsibility to support void functionality.
Or raise an issue against SCSt to improve that test or just leave it empty like I did in the Kinesis Binder:
```
@Test
@Override
public void testClean() {
}
```

If we don't provide such a functionality what is the point to support it?